src/OrderLinesList/OrdersLinesFilterConfig.js

```diff
@@ -50,3 +49,13 @@ export const filterConfig = [
     values: [],
   },
 ];
+
+export const searchableIndexes = [
+  { label: 'Keyword', value: '', makeQuery: term => `(title="${term}*" or contributors="${term}*" or poLineNumber="${term}*" or requester="${term}*" or vendorDetail.vendorAccount="${term}*" or vendorDetail.refNumber="${term}*" or details.productIds="${term}*")` },
```

Comment:
Are these labels translated? I suppose all strings should be tranlsated

Reply:
and please format object

Reply:
fixed